_posts/2018-08-08-survey.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: "Launching the 2018 State of Rust Survey"
+author: The Rust Community Team
+description: "Hearing from you about the third year of Rust"
+---
+
+It's that time again! Time for us to take a look at how the Rust project is doing, and where we should plan for the future. The Rust Community Team is pleased to announce our [2018 State of Rust Survey][survey]! Whether or not you use Rust today, we want to know your opinions. Your responses will help the project understand its strengths and weaknesses, and to establish development priorities for the future.
```

Comment:
"where we should plan for the future" sounds strange with the "how" in the first part of the sentence, and sounds like we're trying to decide a place to meet? "what we should plan for the future" sounds better to me

Reply:
"Your responses will help the project understand its strengths and weaknesses, and to establish development priorities for the future."

I would change this to:

"Your responses will help the project understand its strengths and weaknesses and establish development priorities for the future."

(remove the comma and "to")